contracts/credit/CreditConfiguratorV3.sol

```diff
@@ -277,10 +277,10 @@ contract CreditConfiguratorV3 is ICreditConfiguratorV3, ACLNonReentrantTrait {
             revert AddressIsNotContractException(targetContract); // I:[CC-10A]
         }
 
-        if (
-            targetContract == creditManager || targetContract == creditFacade() || adapter == creditManager
-                || adapter == creditFacade()
-        ) revert TargetContractNotAllowedException(); // I:[CC-10C]
+        address cf = creditFacade();
+        if (targetContract == creditFacade() || adapter == creditFacade()) {
```

Comment:
ah, `cf` is introduced but not used lol, fixed in https://github.com/Gearbox-protocol/core-v3/pull/279/commits/564f07daf0e8c3bb20ae118646fc2909999ddac9